recipes/prefect/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
As the GitHub sources contain this file it can be deleted from the recipe directory (it should also be included in the PyPI tarball if you decide to use that in future).

Reply:
IIUC I need to wait for https://github.com/PrefectHQ/prefect/pull/836 to be in the next release.